packages/pluggableWidgets/accordion-native/CHANGELOG.md

```diff
@@ -5,5 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [2.0.0] Accordion - 2021-9-28
```

Comment:
In web we don't include the widget name, it feels kind of redundant together with the file name. Is there a particular reason you want to inclcude this @wesleysieses ? I just want to make sure CHANGELOGs are aligned across the repo for what it's worth
```suggestion
## [2.0.0] - 2021-9-28
```

Reply:
Initially widget changelogs were generated the same way as module changelogs, and for modules it's useful as those also include the widgets' changelogs, so it's easy to distinguish. 
We recently split changelog generation, so now we can actually leave out the widgets name in the widgets changelogs, while keeping the widgets name in the module changelogs. I will update accordingly :)

Reply:
Makes sense, looks good!